tasks/mrrc-collect-params/README.md

```diff
@@ -0,0 +1,11 @@
+# mrrc-collect-params
```

Comment:
What is MRRC? At the very least it should be defined in the description of the task. I even opened the Jira and that doesn't define it either.

Reply:
MRRC is maven.repository.redhat.com, which is used to host maven artifacts of RedHat Middleware products. This task is one of the whole tasks for the new process to publish content to MRRC.

Reply:
I'll add a simple description of the MRRC in README

Reply:
Cool, thanks.

Reply:
Fixed in new commit